.github/workflows/ci.yml

```diff
@@ -0,0 +1,159 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
We can use these [settings](https://github.com/tracel-ai/burn/blob/main/.github/workflows/test.yml#L3-L28) to configure the triggers. In this way, we reduce the CI burden